providers/aws-ec2/src/main/java/org/jclouds/aws/ec2/AWSEC2Api.java

```diff
@@ -132,4 +133,11 @@
    @Delegate
    Optional<? extends AWSSubnetApi> getAWSSubnetApiForRegion(
            @EndpointParam(parser = RegionToEndpointOrProviderIfNull.class) @Nullable String region);
+
+
+   /**
+    * Provides synchronous access to InternetGateway services.
+    */
+   @Delegate
+   Optional<? extends InternetGatewayApi> getInternetGatewayApi();
```

Comment:
Should we better provide the `InRegion` variant for consistency with other APIs?